WcaOnRails/spec/controllers/registrations_controller_spec.rb

```diff
@@ -169,16 +167,29 @@
       expect(registration.competition_id).to eq competition.id
     end
 
+    it "can re-create registration after it was deleted" do
+      registration = FactoryGirl.create :registration, :accepted, :deleted, competition: competition, user_id: user.id
```

Comment:
Nevermind! I forgot how this works. A registration that is both accepted and deleted is in fact, deleted.